srcpkgs/clex/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'clex'
+pkgname=clex
+version=4.6.patch9
+revision=1
+archs="x86_64 i868 x86_64-musl"
+build_style="gnu-configure"
+configure_args="--prefix=/usr --sysconfdir=/etc --infodir=/usr/share/info --mandir=/usr/share/man --localstatedir=/var"
```

Comment:
Those are default values in void and are already set by the build-style, you can remove the entire line.